usecases/auth/authorization/rbac/authorizer_test.go

```diff
@@ -470,6 +469,149 @@ func TestFilterAuthorizedResourcesLogging(t *testing.T) {
 		"Allowed resources should match input resources")
 }
 
+func TestAuthorizeResourceAggregation(t *testing.T) {
+	// Setup proper logger with hook for testing
+	logger := logrus.New()
```

Comment:
you could use `logrustest "github.com/sirupsen/logrus/hooks/test"`

then the hook is created by default:
```
logger, loggerHook := logrustest.NewNullLogger()
logger.SetLevel(logrus.InfoLevel)
```

Reply:
we discussed internally and agreed to keep it